modules/lotamePanoramaIdSystem.js

```diff
@@ -211,10 +228,17 @@ export const lotamePanoramaIdSubmodule = {
           if (response) {
             try {
               let responseObj = JSON.parse(response);
-              saveLotameCache(KEY_EXPIRY, responseObj.expiry_ts);
+              const shouldUpdateProfileId = !(
+                utils.isArray(responseObj.errors) &&
+                responseObj.errors.indexOf(MISSING_CORE_CONSENT) !== -1
+              );
+
+              saveLotameCache(KEY_EXPIRY, responseObj.expiry_ts, responseObj.expiry_ts);
```

Comment:
should this key-value ever get cleared from the cache?

Reply:
It is used to tell us when we should check the server again so removing is the same as not updating it and there is no practical benefit to explicitly clear this value from the cache